contracts/Conference.sol

```diff
@@ -172,9 +177,50 @@ contract Conference is Destructible, GroupAdmin {
      * @return True if the user is marked as attended by admin.
      */
     function isAttended(address _addr) public view returns (bool){
-        return isRegistered(_addr) && participants[_addr].attended;
+        if (!isRegistered(_addr)) {
+            return false;
+        }
+
+        if (participants[_addr].attended) {
+            return true;
+        }
```

Comment:
In this function you are checking `participants[_addr].attended` first then `attendanceMaps` whereas in `totalAttended` the order is reverse. Would this cause any inconsistent state?

Reply:
No, because I only return if `participants[_addr].attended` is true, if which it won't be if we're using attendance maps.

Reply:
But I could flip the code around, i guess.